src/main/java/domain/LineGenerator.java

```diff
@@ -0,0 +1,26 @@
+package domain;
+
+import java.security.SecureRandom;
+import java.util.ArrayList;
+import java.util.List;
+
+public class LineGenerator {
+    public Line generate(int personCount) {
+        final SecureRandom secureRandom = new SecureRandom();
+        final List<Point> points = new ArrayList<>();
+        Point pastPoint = Point.EMPTY_POINT;
+        for (int count = 1; count < personCount; count++) {
+            boolean randomPoint = secureRandom.nextBoolean();
+            points.add(generateValidatePoint(randomPoint, pastPoint));
+            pastPoint = Point.from(randomPoint);
+        }
+        return new Line(points);
+    }
+
+    private Point generateValidatePoint(final Boolean isLink, final Point pastPoint) {
```

Comment:
```suggestion
    private Point generateValidatePoint(final boolean isLink, final Point pastPoint) {
```
wrapper 클래스는 null이 들어갈수있어서 원시타입을 써주는게 더 안전할것 같아요